test/functional/mining_basic.py

```diff
@@ -13,6 +13,7 @@
 
 from test_framework.blocktools import (
     create_coinbase,
+    NORMAL_GBT_REQUEST_PARAMS,
```

Comment:
Could you point me to where this is defined?

Reply:
`git grep NORMAL_GBT_REQUEST_PARAMS` ;)

Reply:
Oh, I see, I assumed it was introduced in this PR... Resolve pls